docs/main/existing-rhdh.md

```diff
@@ -154,7 +154,7 @@ To include the Notification Plugin append this configuration to the ConfigMap:
         pluginConfig:
           dynamicPlugins:
             frontend:
-              redhat.plugin-signals: {}
+              backstage.plugin-signals: {}
```

Comment:
is this change due to a change in the RHDH 1.4 spec ?

Reply:
The change is due to RHDH 1.4 now supporting default enablement of the signals/notification plugins. I'm not sure in details how it works honestly.

Reply:
cc @batzionb

Reply:
if it's due to 1.4 then I have no concerns.